javaharness/java/arcs/api/PortableJsonParser.java

```diff
@@ -1,18 +1,59 @@
 package arcs.api;
 
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+
 import java.util.Collection;
 
-/** Portable JSON parser that works on either JS or Android environment. */
-public interface PortableJsonParser {
-  PortableJson parse(String json);
+import javax.inject.Inject;
+
+/** Portable JSON parser. */
+public class PortableJsonParser {
```

Comment:
if we are getting rid of the PortableJson abstraction, this class seems redundant.

Reply:
Reverted back.